mibs/hp/ENTITY-STATE-MIB

```diff
@@ -0,0 +1,333 @@
+ENTITY-STATE-MIB DEFINITIONS ::= BEGIN
```

Comment:
Remove this mib please, it's already in the `mibs/` folder.

Reply:
@laf - Removed.